src/android/facebookconnect.xml

```diff
@@ -1,3 +1,5 @@
 <?xml version='1.0' encoding='utf-8'?>
 <resources>
+    <string name="fb_app_id">1614119758827093</string>
+    <string name="fb_app_name">app.hurdlr.com</string>
```

Comment:
this should not be included since it is app-specific (and cordova plugin's are obviously generic and can be used on any app)